### PR TITLE
Jellyfin.JellyfinServer version 10.7.7.0

### PR DESCRIPTION
**Pull request by laedit**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34735)

### CODE REVIEW

File: manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Jellyfin.JellyfinServer
+PackageVersion: 10.7.7.0
```
Comment (jedieaston) [inline manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.installer.yaml:5]:
```suggestion
PackageVersion: 10.7.7
```

File: manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Jellyfin.JellyfinServer
+PackageVersion: 10.7.7.0
```
Comment (jedieaston) [inline manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 10.7.7
```

File: manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: Jellyfin.JellyfinServer
+PackageVersion: 10.7.7.0
```
Comment (jedieaston) [inline manifests/j/Jellyfin/JellyfinServer/10.7.7.0/Jellyfin.JellyfinServer.yaml:5]:
```suggestion
PackageVersion: 10.7.7
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Note that you'll also have to change the folder name since it has to match the version number.

Hot tip to change the folder name: Press "." on your keyboard when you have GitHub open to this PR in your browser, and a web version of VSCode will launch.
 
 Once it loads, find the files you added, and rename the folder. Then use the source control button on the left to commit (it's automatically pushed).
 
 image

**Comment by laedit** [comment]
Thanks for the tip, it is awesome!

**Review by jedieaston — APPROVED** [review]

**Comment by ItzLevvie** [comment]
Looks like pipelines isn't running, you can rerun it by closing and reopening your PR.

**Pull request merged**